### DISCUSSION TITLE
叙一名转专业+休学的大学生经历 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#213] [General]
# 叙一名转专业+休学的大学生经历 - 愧怍的小站

Untitled

https://kuizuo.cn/en/narrate-a-college-student/#comments

**Comment by heywalter** [comment]
加油，关注博客很久了，成功的定义有很多种，敢于尝试就是成功，这些丰富的经历只会让你更强大

**Comment by swx420816530** [comment]
主观上有风险的事，哪怕利润再高，也不会尝试一番，经不起风险，至少在没有”保险”的前提下.(适用于大部分人)(当然也会有人运气很好)

**Comment by luckfunc** [comment]
愧子，你这生活比普通的大学生丰富太多了

**Reply by kuizuo** [reply]
我一直都不认为我是在读大学，我只是借大学的契机来做自己想做的事情。

相比甘愿平淡的人，每日不断反复的生活，又能有什么经历呢。

**Reply by luckfunc** [reply]
Repsect

**Comment by Mas0nShi** [comment]
PATPAT

**Reply by kuizuo** [reply]
不碰爬虫了😭

**Comment by Ashishxp** [comment]
厉害了，我本科毕业快10年了，都没有对技术有过真正的热情只是混口饭吃~加油，你的能力已经超过很多人了，慢慢都会好起来的

**Reply by kuizuo** [reply]
唉，早就没当初那般真正无忧无虑的热情了。

**Comment by BlueOrgreen** [comment]
加油, 愧怍大佬

**Comment by cxOrz** [comment]
这段经历也很精彩了，继续保持热爱！（👮很符合我对这类人的印象hhh

**Reply by kuizuo** [reply]
合法的流氓（bushi

**Comment by schwa1997** [comment]
我最近也正在经历一些事情 你的每句话都写在了我的心里哈哈哈哈 加油吧 很多时候很多事情只是命数 自己的人生怎么过还是得边走边看

**Reply by kuizuo** [reply]
走一步看一步

**Comment by bettercallcyz** [comment]
去年凌晨被开箱喝茶, 抱抱你\(￣︶￣*\))

**Comment by frankelinli** [comment]
你写的“魔改docusaurus教程”，应该是中文圈关于这个主题的第一份，我找了很久，终于发现你写的教程，跟着做，完没实现了。

**Comment by Noah0115** [comment]
以后千万不要触碰法律底线，懂的人自然懂。一起加油，程序员改变世界!

**Comment by imroc** [comment]
精彩！佩服！这个基于docusaurus DIY的网站也很精美，值得借鉴

**Comment by WilmerZhu** [comment]
加油，你其实很厉害，我作为非CS专业的，从你这儿学了不少。君子不器，但还是要爱惜羽毛。经历事情，增加阅历，感觉你已经算悟得很早的了，祝你以后人生顺利！

**Comment by PPsteven** [comment]
通过docusaurus误打误撞的被引流到了你的站点，被博主的故事所吸引。看你的描述，大概率是类似于爬虫这种灰色产业。我之前也写过一些爬虫代码，非盈利，后面觉得对技术上成长也不太大，赚多了容易被喝茶，就没有继续深入。
抛开这些经历不谈，我觉得博主对于休学创业这点有点不理性，像你对技术有追求，最后正常去个大厂结果还是不错的。不过幸运的是，计算机技术是无国界的，你完全可以通过独立开发或者Remote Job实现你的理想，可能打开另一片天地。

**Reply by kuizuo** [reply]
当时的我年少轻狂，极为的任性。是根本不会去想自己做的某些事会带来那些后果，只局限于当下眼前的利益，所以才会想着休学。
现在让我再做选择，我宁可在大学期间混混日子，拿个毕业证，也不会有这么多麻烦事。不过我也庆幸我提前经历了这些，让我成长了些许，变得沉着冷静。

**Comment by frankelinli** [comment]
其实，年轻时多犯点事，多犯点错，是有好处的。

**Comment by mtt-mtt** [comment]
敢于尝试就是胜利加油
